### ISSUE TITLE
Discord Webhook Issue

### ISSUE THREAD

**Issue by Morpheusxeno** [wekan/wekan#2079]
* Did you test in newest Wekan?: Yes
* For new Wekan install, did you configure root-url correctly: Yes, Board works correctly.
* Wekan version: 1.97
* Operating System: Cent os 7
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source): Snap
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first):
* Node Version: Snap
* MongoDB Version: Snap
* ROOT_URL environment variable http(s)://(subdomain).example.com(/suburl): @ root

When adding a discord URL webhook, Nothing happens after submitting the hook and tampering with the board. It accepts the URL webhook. Discord doesnt get any information from the board.
I am not using HTTPS
I have not setup a Mail server with the board.

Edit: Seems like you didn't even read my issue. I didn't say anything about slack.

**Comment by xet7** [comment]
I presume you did not add /slack to end of your webhook URL.

For example, when in Discord channel settings you add webhook
https://discordapp.com/api/webhooks/...

You need to add /slack to the end of URL, to tell that webhook format is slack:
https://discordapp.com/api/webhooks/.../slack

And after that add that /slack including URL to Wekan board, and then make changes to board.

This is already documented here:
https://github.com/wekan/wekan/wiki/Outgoing-Webhook-to-Discord

Works for me.

**Comment by Morpheusxeno** [comment]
https://discordapp.com/api/webhooks/.../discord does not work as a web hook. And when it is input into the board as a web hook, no changes can be made to the board. They instantly get reverted

**Comment by xet7** [comment]
I'm trying to find how to duplicate that non-editable bug.

![outgoing-webhook-discord](https://user-images.githubusercontent.com/15545/50528735-bf59a080-0af8-11e9-8f45-c64b9a39420e.gif)

**Comment by xet7** [comment]
I did read your issue, sorry I did not write clearly enough. I'll try again.

Wekan uses this type of JSON when sending to Outgoing Webhook:
https://github.com/wekan/wekan/wiki/Webhook-data

Discord supports incoming webhooks in different formats, like GitHub, Slack, etc. The incoming format needs to be specified by adding webhook format to end of URL.
https://discordapp.com/developers/docs/resources/webhook#execute-slackcompatible-webhook

Wekan generated webhooks are Slack compatible. Discord does not know anything about Wekan, Rocket.Chat, and other apps that produce Slack compatible Outgoing Webhook format. But using any other format like GitHub etc does not work, because Wekan Outgoing Webhooks are not in that format.

When making Wekan Outgoing Webhook to Rocket.Chat and Slack, there is no need to add anything to Webhook URL when those that is added to Wekan board. Discord in this case has decided to implement multiple Incoming Webhook formats and require specifying format in URL.
